bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategy.scala

```diff
@@ -255,11 +254,10 @@ object SingleProbeStrategy {
    * Calculate the position angle to a target from a specified base position.
    */
   def calculatePositionAngle(base: Coordinates, st: SiderealTarget): Angle = {
-    val ra1    = st.coordinates.ra.toAngle.toRadians
-    val dec1   = st.coordinates.dec.toAngle.toRadians
-    val target = HmsDegTarget.fromSkyObject(st.toOldModel)
-    val ra2    = Angle.fromDegrees(target.getRa.getAs(Units.DEGREES)).toRadians
-    val dec2   = Angle.fromDegrees(target.getDec.getAs(Units.DEGREES)).toRadians
+    val ra1    = base.ra.toAngle.toRadians
+    val dec1   = base.dec.toAngle.toRadians
+    val ra2    = st.coordinates.ra.toAngle.toRadians
+    val dec2   = st.coordinates.dec.toAngle.toRadians
     val raDiff = ra2 - ra1
```

Comment:
I fixed an apparent bug here (introduced in c7268d2) ... the `base` coordinates were never used and the method always returned zero. This was not caught by any test and I don't know what the impact actually is. Any ideas?